pyproject.toml

```diff
@@ -149,3 +149,36 @@ markers = [
     "needs_solver"
 ]
 testpaths = "idaes_examples"
+
+[tool.typos.files]
+extend-exclude = [
+    "*.svg",
+    "*.json",
+    "*.css",
+    "*.yml",
+]
+
+[tool.typos.default.extend-words]
+# Ignore IDAES
+IDAES = "IDAES"
+# Ignore HDA, assumes it's on purpose and not a typo of "had"
+HDA = "HDA" 
+# Ignore Attemp - assume it is abbreviating attemperator
+Attemp = "Attemp"
+# Ignore ba in hexadecimal values, or block names
+ba = "ba"
+# Ignore equil, assumes it's on purpose and not a typo of "equal"
+equil = "equil"
+# Atomic elements
+Nd = "Nd"
+# RIPE specific variable names, remove exceptions if RIPE examples are removed
```

Comment:
Seeing as RIPE has been removed from IDAES, should these just be removed entirely? Do these examples even work at this point?

Reply:
They don't work, which as you pointed out is because the RIPE tool doesn't exist anymore in IDAES. I'm fine with removing them.